week06_deployment/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployment
+
+See README files inside directories for more info about how to run stands.
+
+Run `train_model.py` to generate `vgg16.pt`
+
+Use `client.py` in order to send requests for services 1-4
+Use `client-url.py` in order to send requests for service 5
+
+You can find small dataset of pics in `05_separate_apis/imgs/`
```

Comment:
Maybe 1-2 images per class would be sufficient. An even better solution would be just to link the original dataset (or its relevant part)

Reply:
will add 2 images per class